config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -9,3 +9,23 @@ presubmits:
       - image: golang:1.11
         command:
         - make
+  kubernetes/kubernetes:
+    - name: pull-publishing-bot-build
+      always_run: false
+      skip_report: false
+      optional: true
+      run_if_changed: '^staging/publishing.*$'
+      extra_refs:
+        - org: kubernetes
+          repo: publishing-bot
+          base_ref: master
+          path_alias: k8s.io/publishing-bot
+      spec:
+        containers:
+          - image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-master
+            command:
+              - go
+            args:
+              - run
+              - k8s.io/publishing-bot/cmd/validate-rules
+              - /go/src/k8s.io/kubernetes/staging/publishing/rules.yaml
```

Comment:
IIRC it will be in `/home/prow/go/src/...`
We also need a `path_alias: k8s.io/kubernetes` on the job definition (not in extra refs, the top level field refers to the PR repo checkout)

Reply:
Without the path alias it will be `/home/prow/go/src/github.com/kubernetes/kubernetes` IIRC.

Reply:
👍